packages/kbn-typed-react-router-config/src/create_router.ts

```diff
@@ -99,7 +99,15 @@ export function createRouter<TRoutes extends RouteMap>(routes: TRoutes): Router<
       if (optional) {
         return [];
       }
-      throw new Error(`No matching route found for ${paths}`);
+
+      let errorMessage: string;
+
+      if (paths.length === 1) {
+        errorMessage = `${paths[0]} does not match current path ${location.pathname}`;
+      } else {
+        errorMessage = `None of ${paths.join(', ')} match current path ${location.pathname}`;
```

Comment:
This might be easier to read:

```suggestion
        errorMessage = `None of ${JSON.stringify(paths)} match current path "${location.pathname}"`;
```

(just a thought)

Reply:
Thanks! I think I'm okay with how it is though.